environment-map global scope (#227)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

An environment is named after a class, instance or group identifier (or a
combination thereof).

> An environment MUST be globally unique.

Reviewer: For example, an environment with just `class-id` may denote a specific organization's SKU (product name and specifications), the `instance` field is unique within the `class-id` namespace.

> An environment MUST be globally unique.

Reviewer: ```suggestion
An environment MUST be globally unique.
For example, an environment with just `class-id` may denote a specific organization's SKU (product name and specifications), the `instance` field is unique within the `class-id` namespace.
```

> The combination of values within `environment-map` may be combined so that the combination is globally unique.

Reviewer: ```suggestion
The combination of values within `environment-map` must combine to form a globally unique identifier.
```
